src/src/MatrixImplementation.java

```diff
@@ -30,7 +31,14 @@ public int[][] multiplyMatrices(int[][] matrix1, int[][] matrix2){
       }
     }
     return resultantMatrix;
-  }
+  }catch(IllegalArgumentException e){
+	  System.out.print("Error: " + e.getMessage());
+	  e.printStackTrace();
+	  // would this errors pass process boundaries? 
```

Comment:
I dont think it would since this would catch the Exception and then end the program.